pkg/virt-launcher/virtwrap/cli/libvirt.go

```diff
@@ -313,6 +313,7 @@ func (l *LibvirtConnection) GetDomainStats(statsTypes libvirt.DomainStatsTypes,
 		stat.CPUMapSet = true
 
 		list = append(list, stat)
+		domStat.Domain.Free()
```

Comment:
I think this will cause a double free. this free occurs in the defer function already, right?